### ISSUE TITLE
Need to update chat widget JS file on CDN/cloud

### ISSUE THREAD

**Issue by rounakraj** [samtecspg/articulate#972]
Hi, 
I am using Articulate Master Branch. During Web Demo testing when i am opening the web demo link in different browsers, the chat session is shared. Both are getting the replies from the bot.

Attached screenshot:
<img width="343" alt="Screenshot 2019-12-19 at 3 02 57 PM" src="https://user-images.githubusercontent.com/6190350/71162127-a8afda80-2270-11ea-896a-327407b1ff06.png">

![image](https://user-images.githubusercontent.com/6190350/71162215-d2690180-2270-11ea-9fe4-b37000737646.png)

As you can see in screen shot from 2 different browser the chat reply is shared.

**Comment by rounakraj** [comment]
Same is observed with web chat also, session seems to be shared.
Attached screenshot.
<img width="417" alt="Screenshot 2019-12-19 at 5 41 49 PM" src="https://user-images.githubusercontent.com/6190350/71172717-fe8f7d00-2286-11ea-8385-7b65846e42d3.png">
<img width="400" alt="Screenshot 2019-12-19 at 5 41 41 PM" src="https://user-images.githubusercontent.com/6190350/71172718-fe8f7d00-2286-11ea-884b-b6616e7f10ae.png">

**Comment by wrathagom** [comment]
I’ve tested the changes and I find it likely that you just have a docker cache problem
So you’re not actually running the latest? What steps did you take to update?

**Comment by wrathagom** [comment]
Actually, looking over your images I don’t see anything wrong.

How did you respond to the hello how are you?

**Comment by rounakraj** [comment]
> I’ve tested the changes and I find it likely that you just have a docker cache problem
> So you’re not actually running the latest? What steps did you take to update?

Ok I just did a git pull and then docker compose up. How do we clear docker cache?

**Comment by rounakraj** [comment]
> Actually, looking over your images I don’t see anything wrong.
> 
> How did you respond to the hello how are you?

Above two snapshot of chat are from different session. In the second session i asked  hi to which the bit replied hello how are you. These response also comes in 2nd chat window.

**Comment by wrathagom** [comment]
Since your on master the names of the images don’t change. Which means that docker already thinks it’s has an image to use.

```
docker-compose down
docker-compose pull
docker-compose up -d
```

**Comment by rounakraj** [comment]
Thanks i will try these and let you know.

**Comment by rounakraj** [comment]
I tried the above steps and could see that it pulled api.
But the issue still persists.
Check the attached screen shot:
<img width="424" alt="Screenshot 2019-12-19 at 7 19 38 PM" src="https://user-images.githubusercontent.com/6190350/71178641-a1022d00-2294-11ea-83db-9279bc089277.png">
This is the first session where i send hi.
The response comes in both the sessions.
<img width="424" alt="Screenshot 2019-12-19 at 7 19 43 PM" src="https://user-images.githubusercontent.com/6190350/71178722-c8f19080-2294-11ea-8c76-ec02be5c804b.png">

**Comment by wrathagom** [comment]
@dakavisa was this fix for this on the API side or in the UI?

@rounakraj have you cleared the browser cache as well? To make sure you loaded the newest chat component.

**Comment by wrathagom** [comment]
@dcalvom @dakavisa does the commit to the chat widget automatically updated the source files that they load in the page? Or do we manually publish those someplace?

**Comment by rounakraj** [comment]
> @dakavisa was this fix for this on the API side or in the UI?
> 
> @rounakraj have you cleared the browser cache as well? To make sure you loaded the newest chat component.

I tried clearing browser cache as well.

**Comment by wrathagom** [comment]
Okay, my guess is that you're not building the chat widget right? But rather just using the files stored in the cloud? I don't think those files got updated with @dakavisa most recent changes, when he comes online hopefully he'll see this and update those files.

**Comment by dakavisa** [comment]
@rounakraj did you use Articulate directly to generate the chat widget connection? if that is the case could you please paste here the following section of the generated html to see which files are you pulling (head section)

![image](https://user-images.githubusercontent.com/53984674/71185015-3fb06e80-2240-11ea-90d4-1fc9ff9b3438.png)

**Comment by rounakraj** [comment]
@dakavisa  Yes i used articulate directly to generate chat widget connection.
<img width="964" alt="Screen Shot 2019-12-20 at 12 40 16 AM" src="https://user-images.githubusercontent.com/6190350/71202010-4f23cc00-22c1-11ea-8879-b21437c16700.png">

**Comment by dakavisa** [comment]
Web demo was sharing sessions among browsers, it was fixed with 177a7e8fa65fc4809b7b5a1a8550feb33df8850e, chat widget seems to work across different browsers (with different tabs it will be the same conversation)

**Comment by rounakraj** [comment]
Thanks will check.
